### DISCUSSION TITLE
Database wont start using helm

### DISCUSSION THREAD

**Discussion by ArtRogers** [nextcloud/all-in-one#5169] [Questions] [answered]
### Steps to reproduce
1. kubernetes using microk8s and nfs-csi storage class
2. execute the helm per your instructions (https://github.com/nextcloud/all-in-one/tree/main/nextcloud-aio-helm-chart)
3. db fails to start

### Expected behavior 
all pods should start

### Actual behavior 
nextcloud-aio-database pod does not start; all else do
'kubectl describe' shows error on the chown -R 999:999
     Command:
      chown
      999:999
      -R
      /nextcloud-aio-database
      /nextcloud-aio-database-dump
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error

I can find no issues in NFS

### Host OS 
Ubuntu 22.04.4 LTS, 6.8.0-40-generic #40~22.04.3-Ubuntu SMP PREEMPT_DYNAMIC

#### Docker run command or docker-compose file that you used
used  helm
#### Nextcloud AIO version 
The interface is not up. The helm was updated yesterday, 8/20/24

#### Other valuable info 
This looks similar to https://github.com/nextcloud/all-in-one/issues/2466, but is occurring today with the latest helm.
I have STORAGE_CLASS: nfs-csi configured in the values.yaml
NFS exports is simple: /srv/nfs 192.168.1.0/24(rw,sync,no_subtree_check)

I have attempted clean reinstall and experience same. 
The data subdir is created in the nextcloud-aio-database persistent volume

**Comment by szaimen** [comment, marked as answer]
Hi,

> 'kubectl describe' shows error on the chown -R 999:999
> Command:
> chown
> 999:999
> -R
> /nextcloud-aio-database
> /nextcloud-aio-database-dump
> State: Waiting
> Reason: CrashLoopBackOff
> Last State: Terminated
> Reason: Error

looks like you found the issue then

**Reply by ArtRogers** [reply]
I don’t know how to get past it. Any thoughts on that?
It stops there every time.

**Reply by szaimen** [reply]
Maybe root access is not permitted on your nfs server? otherwise no clue...
